server/models/student.js

```diff
@@ -0,0 +1,34 @@
+let mongoose = require('mongoose');
+let activity = require('activity.js')
+let Schema = mongoose.Schema;
+
+let StudentSchema = new Schema({
+    id:
+    {
+        type: Number,
+        required: true
+    },
+    activities: [
+        {
+            type: activity
+        }],
+    img_url: {
+        type: String
```

Comment:
It looks like we can use a url type, which will keep our code more robust:
https://stackoverflow.com/questions/47134609/how-to-store-url-value-in-mongoose-schema